pkg/virt-handler/vm.go

```diff
@@ -586,10 +590,17 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 						if err != nil {
 							return err
 						}
-						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP {
+
+						shouldCopy := !stringSlicesEqual(existingInterfaceStatusByName[domainInterface.Alias.Name].IPs, iface.PodIPs[:])
+						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP ||
+							shouldCopy {
 							newInterface.Name = domainInterface.Alias.Name
 							newInterface.IP = iface.PodIP
-							newInterface.IPs = []string{iface.PodIP}
+							newInterface.IPs = []string{}
+
+							for _, ip := range iface.PodIPs {
```

Comment:
same

Reply:
lets continue the discuss at 
https://github.com/kubevirt/kubevirt/pull/3958/files/5d1e3b364c04018d3c14acb080f5ed3846fe002b#r469236295